src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/packageinfo/test4/package-info.java

```diff
@@ -1,5 +1,5 @@
 //for test there's warning when single line comment isn't separated from PACKAGE_DEF by line
-//violation is expected after this line
+//*violation is expected after this line
```

Comment:
Please explain

Reply:
This was to kill this mutation: http://rveach.no-ip.org/checkstyle/regression/pitest-reports/30/pitest-whitespace/com.puppycrawl.tools.checkstyle.checks.whitespace/EmptyLineSeparatorCheck.java.html#org.pitest.mutationtest.report.html.SourceFile@7d2fc711_570

`JavadocUtil.isJavadocComment` doesn't re-check if the AST given to it is actually a javadoc or not. All the utility does is ensure the text of the AST given to it starts with a `*`. To kill the mutation, I had to give it a block comment that starts with `*` in a package-info file to show it that it shouldn't treat everything as a javadoc and kill the mutation.

Reply:
> I had to give it a block comment

Single line comment.

> comment that starts with * , it shouldn't treat everything as a javadoc 

Please add this to the of comments, to self document such unusual test

Reply:
Done.